tc/sgx/trusted_worker_manager/enclave/CMakeWorkloads.txt

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Create list of supported workloads
```

Comment:
Better to clear the list first or set it to empty before adding elements
`set(SUPPORTED_WORKLOADS_LIST  "")`

Reply:
Done